### DISCUSSION TITLE
[Idea] Add an option so that SNMP inventory does not create a new location

### DISCUSSION THREAD

**Discussion by JulienRemi** [glpi-project/glpi-agent#713] [Ideas]
### Your idea

Hi, doing a SNMP inventory will create locations based on the SNMP location fields. It would be nice to have an option so that the location isn't created at all or only when the agent doing the inventory already have one.  

Currently, new location will be created but they are empty because the inventoried item is placed under the location of the agent (which is a good behavior)

I think it's more server side related, but there is no option to open discussion or create feature request on the glpi-inventory-plugin repo.

**Comment by g-bougard** [comment]
Hello @JulienRemi 

this is definitively a server-side problem.

Did you have check the Localization rules in Administration panel ? It seems the rules actions available won't help you, but maybe you can just ask for new actions and I guess new criteria should be available to match on device type also.

Anyway the right place to discuss for such server-side idea is here: https://glpi.userecho.com/
Check first all available ideas if anyone looks like yours. If you found one, upvote and eventually comment it. Of course, you can create any new one there.
